conf.yaml

```diff
@@ -123,6 +123,16 @@ targets:
     hosts:
       - discordapp.com
     twitter: "@discordapp"
+  Docker:
+    href: https://www.docker.com/
+    hosts:
+      - docker.com
+      - hub.docker.com
+      - registry-1.docker.io
+      - registry.docker.io
+    icon: "fa:docker"
```

Comment:
You specified this key twice. This is not how YAML works.

Generally I prefer having a font-awesome icon instead of copying their picture. That way we (hopefully) avoid weird license constraints.

Reply:
i know, i override the `icon` entry, because the docker font-awesome icon is added in the newest version https://fontawesome.com/icons/docker?style=brands within the **brands** part.

Reply:
Docker is at [Apache 2.0 Licence](https://www.docker.com/components-licenses) - and [here](https://www.docker.com/brand-guidelines) is the brand guidelines with the logo i downloaded from